### ISSUE TITLE
Cookie usage 

### ISSUE THREAD

**Issue by adelatuduce** [jamesdbloom/mockserver-client-ruby#4]
I am trying to run multiple tests in parallel and I need to obtain independent responses for each request made through MockServer.

I am using the gem with following request:
` client = initialize_mock
    expectation = expectation do |e|
      e.request do |request|
        request.method = 'POST'
        request.path = '/transactions'
        request.cookies << cookie('sessionId', SecureRandom.hex)
      end

      dummy_response = mock_tb_response(successful)

      e.response do |response|
        response.status_code = 201
        response.body = dummy_response.to_json
      end
    end

    # puts expectation.to_yaml
    client.register(expectation)
  end

def initialize_mock
    client = MockServerClient.new(MOCK_SERVER[:host], MOCK_SERVER[:port])
    client.logger = Logger.new("mockserver_logs.log")
    client
  end

def mock_tb_response(successful)
    tb_id = Faker::Number.between(1, 1_000_000)
    successful ? { id: tb_id, v: '2.0.0' } : {}
  end
` 
When I add the `request.cookies << cookie('sessionId', SecureRandom.hex)` part, I can't seem to get a response and my request fails with HTTP 500. If I remove it, the response comes as expected.

What am I doing wrong?

Thank you.

**Comment by jamesdbloom** [comment]
The Ruby client has been completely rewritten as a hand-written idiomatic library (v6.0.0). Cookie handling is now supported through the standard request/response model.

This repository has been merged into the MockServer monorepo: https://github.com/mock-server/mockserver-monorepo

The new client lives in `mockserver-client-ruby/`. Please file any future issues against the monorepo.
